src/dom/DomEvent.js

```diff
@@ -77,7 +77,10 @@ L.DomEvent = {
 		if (L.Browser.pointer && type.indexOf('touch') === 0) {
 			this.addPointerListener(obj, type, handler, id);
 
-		} else if (L.Browser.touch && (type === 'dblclick') && this.addDoubleTapListener) {
+		} else if (L.Browser.touch && (type === 'dblclick') && this.addDoubleTapListener &&
+		           !(L.Browser.pointer && L.Browser.chrome) ) {

```

Comment:
This makes chrome>55 ignore the addDoubleTapListener hack and rely on native `dblclick`s.